Rollbar/Deploys/RollbarDeployClient.cs

```diff
@@ -0,0 +1,198 @@
+﻿namespace Rollbar.Deploys
+{
+    using Newtonsoft.Json;
+    using Rollbar.Common;
+    using Rollbar.Diagnostics;
+    using System;
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    internal class RollbarDeployClient
+    {
+
+        private readonly RollbarConfig _config;
+        private readonly HttpClient _httpClient;
+
+        public RollbarDeployClient(RollbarConfig config, HttpClient httpClient = null)
+        {
+            Assumption.AssertNotNull(config, nameof(config));
+
+            this._config = config;
+
+            this._httpClient = httpClient;
+
+            //var header = new MediaTypeWithQualityHeaderValue("application/json");
```

Comment:
We can get rid of these commented out lines.

Reply:
done. thanks!